.circleci/config.yml

```diff
@@ -40,12 +40,12 @@ commands:
             cd && touch $BASH_ENV
 
             if [ "${CIRCLE_BRANCH}" == "master" ]; then
-              echo 'export INFRA_PATH=iac/conf/websites/prod/beta.cosmoswallet.io' >> $BASH_ENV
-              echo 'export DEPLOYMENT_ENV=beta.cosmoswallet.io' >> $BASH_ENV
+              echo 'export INFRA_PATH=iac/conf/websites/prod/lunie.io' >> $BASH_ENV
+              echo 'export DEPLOYMENT_ENV=lunie.io' >> $BASH_ENV
               echo 'export GIT_BRANCH=${CIRCLE_BRANCH}' >> $BASH_ENV
             else
-              echo 'export INFRA_PATH=iac/conf/websites/dev/${CIRCLE_PROJECT_REPONAME}-staging.interblock.io' >> $BASH_ENV
-              echo 'export DEPLOYMENT_ENV=${CIRCLE_PROJECT_REPONAME}-staging.interblock.io' >> $BASH_ENV
+              echo 'export INFRA_PATH=iac/conf/websites/dev/beta.lunie.io' >> $BASH_ENV
+              echo 'export DEPLOYMENT_ENV=beta.lunie.io' >> $BASH_ENV
```

Comment:
this should be staging.lunie.io, right?

Reply:
Not quite yet. That value is just a key in the AWS parameter store. I haven't changed it yet but will do so sometime soon. For now it's good.